app/src/main/java/chat/rocket/android/app/DateTimeHelper.kt

```diff
@@ -37,7 +37,7 @@ object DateTimeHelper {
     fun getDate(localDateTime: LocalDateTime, context: Context): String {
         val localDate = localDateTime.toLocalDate()
         return when (localDate) {
-            today -> formatLocalTime(localDateTime.toLocalTime())
+            today -> "Today " + formatLocalTime(localDateTime.toLocalTime())
```

Comment:
Please avoid using hardcoded strings. Add that to strings resource files for all languages.